_pages/documentation/general_docs/gpu_models/amdgpu.md

```diff
@@ -1,44 +1,44 @@
 ---
 layout: documentation
-title: GCN3 GPU model
+title: AMD GPU model
 doc: gem5 documentation
 parent: gpu_models
-permalink: /documentation/general_docs/gpu_models/GCN3
+permalink: /documentation/general_docs/gpu_models/vega
 ---
 
-# **GCN3**
+# **AMD GPU model**
 
 Table of Contents
 
 1. [Using the model](#Using-the-model)
 2. [ROCm](#ROCm)
 3. [Documentation and Tutorials](#Documentation-and-Tutorials)
 
-The GCN3 GPU is a model that simulates a GPU at the ISA level, as opposed to the intermediate language level. This page will give you a general overview of how to use this model, the software stack the model uses, and provide resources that detail the model and how it is implemented.
+The AMD GPU is a model that simulates a GPU at the VEGA ISA level, as opposed to the intermediate language level. This page will give you a general overview of how to use this model, the software stack the model uses, and provide resources that detail the model and how it is implemented.
 
 ## **Using the model**
 
-Currently, the GCN3 GPU model in gem5 is supported on the stable and develop branch.
+Currently, the AMD GPU model in gem5 is supported on the stable and develop branch.
```

Comment:
I don't like this change, as it seems like a tautology.  Before it was saying a specific version was supported.  Now it is just saying AMD GPUs run in gem5.

Reply:
I was originally just replacing with vega but i thought not many folks would know what that was, so switched to AMD.  I want to make clear it's different from the (No)Mali model the ARM folks have. I need to put AMD in because reasons, so  perhaps "AMD Vega" would be better?

Reply:
yeah I think this would be better